pkg/console/subresource/configmap/brand_custom_logo_test.go

```diff
@@ -0,0 +1,36 @@
+package configmap
+
+import (
+	"testing"
+
+	"github.com/go-test/deep"
+)
+
+func TestIsLikelyCommonImageFormat(t *testing.T) {
```

Comment:
Data here is generated from a simple image:

gif
![hi](https://user-images.githubusercontent.com/280512/59930366-c8289080-9410-11e9-883d-4f3526c42c17.gif)

jpg
![hi](https://user-images.githubusercontent.com/280512/59930371-c9f25400-9410-11e9-9d0d-31d3e7a1e25c.jpg)

png
![hi](https://user-images.githubusercontent.com/280512/59930394-d080cb80-9410-11e9-8074-cc7d1139d3ca.png)

`oc create configmap hi-png --from-file ~/Desktop/imgs/hi.png -n openshift-config` && `oc get configmap hi-png -o yaml` to see what gets stored in `binaryData` when a configmap is created.